### PR TITLE
Keith.cwp pr2

### PR DESCRIPTION
**Pull request by ace-contributor**
## Description
- `ethereum/commonwealth/api`

## Fixed things
- client/scripts/app.ts
   Convert and move loading from `selectCommunity` to `selectChain`
- Get Commonwealth chain initialization working in
   `ethereum/commonwealth/api`
- Controller connects to the API for the contracts, pulling live data from the testnet

### CODE REVIEW

File: client/scripts/app.ts [hunk 1]
```diff
@@ -157,7 +158,7 @@ export function handleUpdateEmailConfirmation() {
   }
 }
 
-export async function selectCommunity(c?: CommunityInfo, node?: NodeInfo): Promise<boolean> {
+export async function selectCommunity(c?: CommunityInfo): Promise<boolean> {
```
Comment (zakhap) [inline client/scripts/app.ts:161]: Why did you need to do this?

File: client/scripts/app.ts [hunk 2]
```diff
@@ -215,9 +204,11 @@ export async function selectNode(n?: NodeInfo, deferred = false): Promise<boolea
   }
 
   // Check for valid chain selection, and that we need to switch
-  if (app.chain && n === app.chain.meta) {
+  if (app.chain && n === app.chain.meta) 
+  {
     return;
   }
+
```
Comment (zakhap) [inline client/scripts/app.ts:211]: Unnecessary new lines

File: client/scripts/controllers/chain/ethereum/commonwealth/adapter.ts [hunk 3]
```diff
@@ -33,18 +33,18 @@ export default class Commonwealth extends IChainAdapter<EthereumCoin, EthereumAc
...
-      '0x4C7804bf331183adad98Af5AfBdA5F27A3E84e8C', // this.meta.address,
+      '0x487f2D9f9427bBBC97cDc2F77Ab3083ea82b2496', // this.meta.address,
       this.chain.api.currentProvider as any
-    );
+    );    
     await api.init();
     this.chain.CommonwealthAPI = api;
     // await this.chain.initEventLoop();
-    await super.initApi();
+    // await super.initApi();
```
Comment (zakhap) [inline client/scripts/controllers/chain/ethereum/commonwealth/adapter.ts:42]: Why no initialization of API?

File: client/scripts/controllers/chain/ethereum/commonwealth/protocol.ts [hunk 4]
```diff
@@ -174,16 +177,16 @@ export default class CommonwealthProtocol {
...
+    // const res = await $.post(`${app.serverUrl()}/cw/back-project`, {
+    //   backer,
+    //   amount,
+    //   projectHash,
+    //   auth: true,
+    //   jwt: app.user.jwt
+    // });
+    // if (res['status'] === 'Success') {
+    //   await this.updateState();
+    // }
```
Comment (zakhap) [inline client/scripts/controllers/chain/ethereum/commonwealth/protocol.ts:189]: This and lots of the protocol controller post routes seem to be commented out. Why?

File: client/scripts/views/components/sidebar/cw_protocol.ts [hunk 5]
```diff
@@ -1,136 +0,0 @@
...
-            fluid: true,
-            disabled: vnode.state.initializing,
-            label: vnode.state.initializing ? 'Connecting...' : app.chain.deferred
-              ? 'Connect to chain' : m(ChainStatusIndicator),
-          }),
-        }),
-      ]);
-  }
-};
-
```
Comment (zakhap) [inline client/scripts/views/components/sidebar/cw_protocol.ts:136]: Why is this whole file removed? Is this logic distributed elsewhere?

File: server/scripts/resetServer.ts [hunk 6]
```diff
@@ -281,16 +281,16 @@ const resetServer = (models): Promise<number> => {
...
+        // models.Chain.create({
+        //   id: 'ethereum-local',
+        //   network: 'ethereum',
+        //   symbol: 'ETH',
+        //   name: 'Ethereum (local)',
+        //   icon_url: '/static/img/protocols/eth.png',
+        //   active: true,
+        //   type: 'chain',
+        //   base: 'ethereum',
+        // }),
```
Comment (zakhap) [inline server/scripts/resetServer.ts:293]: Why remove this?

**Review by zakhap — CHANGES_REQUESTED** [review]

File: client/scripts/app.ts (hunk 1 above)
Comment (ace-contributor) [reply client/scripts/app.ts:161]: i previously used node as argument in selectCommunity function because cw-protocol was offchain community. To change network for CWP, i needed to select Node in selectCommunity funcation. but since i updated it to Chain, we don't need to change Node in selectCommunity.

File: client/scripts/app.ts (hunk 2 above)
Comment (ace-contributor) [reply client/scripts/app.ts:211]: fixed

File: client/scripts/controllers/chain/ethereum/commonwealth/adapter.ts (hunk 3 above)
Comment (ace-contributor) [reply client/scripts/controllers/chain/ethereum/commonwealth/adapter.ts:42]: forgot it, fixed now.

File: client/scripts/controllers/chain/ethereum/commonwealth/protocol.ts (hunk 4 above)
Comment (ace-contributor) [reply client/scripts/controllers/chain/ethereum/commonwealth/protocol.ts:189]: since we use real chain data, we will not use local API test data now.

File: server/scripts/resetServer.ts (hunk 6 above)
Comment (ace-contributor) [reply server/scripts/resetServer.ts:293]: fixed
